packages/p2p/src/components/my-ads/ad-status.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { observer } from 'mobx-react-lite';
+import { Text } from '@deriv/components';
+import { Localize } from 'Components/i18next';
+import './ad-status.scss';
+
+const AdStatus = ({ days_until_archive, is_active }) => {
+    if (!is_active) {
+        return (
+            <div className='ad-status--inactive'>
+                <Text color='loss-danger' line_height='s' size='xs' weight='bold'>
+                    <Localize i18n_default_text='Inactive' />
+                </Text>
+            </div>
+        );
+    }
+
+    if (is_active && days_until_archive === 1) {
```

Comment:
Do we need to check for `is_active`? since if it's not truthy it will be returned in line 9